console/coloring.rst

```diff
@@ -46,7 +46,9 @@ It is possible to define your own styles using the
     $output->writeln('<fire>foo</>');
 
 Any hex color is supported for foreground and background colors. Besides that, these named colors are supported:
-``black``, ``red``, ``green``, ``yellow``, ``blue``, ``magenta``, ``cyan`` and ``white``.
+``black``, ``red``, ``green``, ``yellow``, ``blue``, ``magenta``, ``cyan``, ``white``,
+``gray``, ``bright-red``, ``bright-green``, ``bright-yellow``, ``bright-blue``,
+``bright-magenta``, ``bright-cyan`` and ``bright-white``.
 
 .. versionadded:: 5.2
```

Comment:
Please add a versionadded:: 5.3 below this one:

Support for bright colors was introduced in Symfony 5.3.

Thanks

Reply:
Added while merging.